src/pages/TermsOfService.js

```diff
@@ -314,12 +314,12 @@ const TermsOfServicePage = () => {
     }
   }
 
-  return div({ style: { padding: '1rem', minHeight: '100%', display: 'flex', justifyContent: 'center', alignItems: 'center' } }, [
+  return div({ role: 'main', style: { padding: '1rem', minHeight: '100%', display: 'flex', justifyContent: 'center', alignItems: 'center' } }, [
     backgroundLogo,
-    div({ style: { backgroundColor: 'white', borderRadius: 5, width: 800, padding: '2rem', boxShadow: Style.standardShadow } }, [
-      div({ style: { color: colors.dark(), fontSize: 38 } }, ['Terra Terms of Service']),
+    div({ style: { backgroundColor: 'white', borderRadius: 5, width: 800, maxHeight: '100%', padding: '2rem', boxShadow: Style.standardShadow } }, [
+      h1({ style: { color: colors.dark(), fontSize: 38, fontWeight: 400 } }, ['Terra Terms of Service']),
       needToAccept && div({ style: { fontSize: 18, fontWeight: 600 } }, ['Please accept the Terms of Service to continue.']),
-      div({ style: { maxHeight: 400, overflowY: 'auto', lineHeight: 1.5, marginTop: '1rem', paddingRight: '1rem' } }, [
+      div({ style: { height: '50vh', overflowY: 'auto', lineHeight: 1.5, marginTop: '1rem', paddingRight: '1rem' } }, [
```

Comment:
I wonder if I'm the only person who is in love with the availability of the vx length specifiers.